mime.go

```diff
@@ -0,0 +1,20 @@
+package viamrtsp
+
+import (
+	"bytes"
+	"image"
+	"image/jpeg"
+
+	"go.viam.com/rdk/components/camera"
+	rutils "go.viam.com/rdk/utils"
+)
+
+func encodeToJPEG(img image.Image) ([]byte, camera.ImageMetadata, error) {
+	buf := new(bytes.Buffer)
+	if err := jpeg.Encode(buf, img, nil); err != nil {
```

Comment:
I think in RDK we do quality 75

https://github.com/viamrobotics/rdk/blob/567b1abdc2f72008ec4f602c6e3af0ac3d6b3143/rimage/image_file.go#L323

Reply:
thanks good to know. looks like 75 is the default anyhow

https://pkg.go.dev/github.com/go-oss/image/jpeg#section-readme